cds/modules/previewer/templates/cds_previewer/macros/player.html

```diff
@@ -1,10 +1,16 @@
 {% macro theo_player(obj, theo_config, record=None, video_classes='', append_to_element=None, video_source=None, log_media_views_url=None, embed_config=None) -%}
+{%- if embed_config.responsive %}
+      {%- set video_classes = video_classes + " embed-responsive-background" %}
+  {% endif %}
   {% if config.THEO_LICENCE_KEY %}
     <script type='text/javascript' src='//cdn.theoplayer.com/dash/{{ config.THEO_LICENCE_KEY }}/THEOplayer.js'></script>
     <script type="text/javascript">
 
       var element = document.createElement('div');
       element.className = 'video-js theo-social-available theoplayer-skin theo-seekbar-above-controls {{ video_classes }}';
+      if (/(MSIE|Trident\/|Edge\/)/i.test(window.navigator.userAgent)){
```

Comment:
Does this mean that also IE before edge works?

Reply:
It seems yes, but I've tested is as an isolated code, because TheoPlayer license does not work for VirtualBox where I tried to test the player. So I need to test it in detail on qa.